### DISCUSSION TITLE
Machine Learning Compilation - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#209] [Announcements]
# Machine Learning Compilation - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E7%B3%BB%E7%BB%9F/MLC/

**Comment by zhaoM0** [comment]
这个真的厉害，就是太鸡儿难了

**Reply by imsubei** [reply]
老哥四川的是吧 hh

**Comment by ruxiliang** [comment]
“国内外此前还没有为这个方向专门开设的相关课程”
严格来说是错误的，uw之前有个类似的课程https://dlsys.cs.washington.edu/，但陈天奇的这门课应该是第一门资料完全的dlsys的课

**Reply by songmuhan** [reply]
这个的instructor也是陈天奇😂

**Reply by wmhst7** [reply]
陈天奇去CMU当AP后UW这个课也没人上了😂

**Reply by wmhst7** [reply]
没人教了*

**Comment by ruxiliang** [comment]
是的，其实就是uw这本科的pro max版本）

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: FkArch ***@***.***>
发送时间: Tuesday, October 11, 2022 7:18:05 PM
收件人: PKUFlyingPig/cs-self-learning ***@***.***>
抄送: ruxiliang ***@***.***>; Comment ***@***.***>
主题: Re: [PKUFlyingPig/cs-self-learning] Machine Learning Compilation - CS自学指南 (Discussion #209)


这个的instructor也是陈天奇😂

**Comment by antonia0912** [comment]
自荐补充 [TVM 完整中文文档](https://tvm.hyper.ai/)，也是由 MLC 社区志愿者翻译和维护的

**Comment by itsuitsuki** [comment]
请问学这个需要以编译原理为前置吗

**Comment by zhang-yuhui** [comment]
这个作业只有一个吗？没找到其他的
